### DISCUSSION TITLE
DTI Brain Extraction Visualization Issue

### DISCUSSION THREAD

**Discussion by lrokos** [McIntosh-Lab/tvb-ukbb#173] [Help]
Hi, I'm having issues with visualizing the DTI brain extraction mask for one of my datasets - I think it might just be a contrast/brightness issue?

**Comment by justinliuyuwang** [comment]
Great question. I assume your dMRI underlay has poor contrast.  I would suggest the following:

1. Load your underlay in fsleyes, in this case /dMRI/dMRI/data_B0.nii.gz 
2. Tune the brightness and contrast to your liking
3. [Show command line arguments for your current view](https://open.win.ox.ac.uk/pages/fsl/fsleyes/fsleyes/userdoc/command_line.html#generating-command-line-arguments)
4. Note the brightness and contrast values in that overlay command line argument
5. In the relevant calls to lightbox.sh in tvb_bb_QC.sh, add -b and -c flags, each followed by the noted brightness and contrast values, respectively.
